### PR TITLE
creating edit profile page

### PR DESCRIPTION
**Pull request by Abdulrahimch**
### What this PR does (required):
- Ceate Edit profile page and make it responsive.

dependencies: 
1-"@date-io/date-fns": "1.3.13"
2-"@material-ui/pickers": "3.3.10"
3-"date-fns": "2.25.0"

### CODE REVIEW

File: client/src/pages/EditProfile/EditProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  const [lastName, setLastName] = useState<string>('');
+  const [gender, setGender] = useState<string>('male');
+  const [birthDate, setBirthDate] = useState<Date | null>(new Date('1988-05-15'));
+  const [email, setEmail] = useState<string>('');
+  const [phone, setPhone] = useState<string>('');
+  const [address, setAddress] = useState<string>('');
+  const [desc, setDesc] = useState<string>('');
+
+  return (
+    <React.Fragment>
```
Comment (ezeilo-su) [inline client/src/pages/EditProfile/EditProfile.tsx:22]: You could also destructure this `Fragment` from `'react'` in line 1

**Review by ezeilo-su — COMMENTED** [review]
Your UI looks good! Good luck with other tickets.

File: client/src/pages/EditProfile/EditProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                        setDesc(event.target.value);
+                      }}
+                    />
+                  </Grid>
+                </Grid>
+                <Grid item container justify="flex-end">
+                  <Button
+                    variant="contained"
+                    size="large"
+                    style={{
```
Comment (rajivtitus) [inline client/src/pages/EditProfile/EditProfile.tsx:187]: Please avoid the usage of inline-styles, you can use a CSS class instead :)

**Comment by rajivtitus** [comment]
@Abdulrahimch Good usage of Material UI! I've left some comments for you to address. Alongside addressing the comments, please also complete the following:
- connect the relevant issue when making PRs
- add an appropriate images/videos to demonstrate feature 
Thank you! 😄

**Pull request closed without merging**